coverage/PENDING_REVIEW/intel/0467-COVERAGE-brw-eu-emit-1366/cov-nested-loops-global-loop-counter-do-while-accumulate-float.amber

```diff
@@ -0,0 +1,660 @@
+# #define _int_2 _GLF_uniform_int_values[2]
+# #define _int_10 _GLF_uniform_int_values[3]
+# #define _float_1_0 _GLF_uniform_float_values[0]
+# #define _float_0_0 _GLF_uniform_float_values[1]
+# #define _float_28_0 _GLF_uniform_float_values[2]
+#
+# precision highp int;
+#
+# precision highp float;
+#
+# // Contents of _GLF_uniform_float_values: [1.0, 0.0, 28.0]
+# layout(set = 0, binding = 0) uniform buf0 {
+#     float _GLF_uniform_float_values[3];
+# };
+# // Contents of _GLF_uniform_int_values: [1, 0, 2, 10]
+# layout(set = 0, binding = 1) uniform buf1 {
+#     int _GLF_uniform_int_values[4];
+# };
+# const int _GLF_global_loop_bound = 100;
+#
+# int _GLF_global_loop_count = 0;
+#
+# layout(location = 0) out vec4 _GLF_color;
+#
+# void main()
+# {
+#     float f = _float_0_0;
+#
+#     for(int i = _int_0; i < _int_1; i++)
+#     {
+#         for(int i = _int_0; i < _int_1; i++)
+#         {
+#             for(int i = _int_0; i < _int_1; i++)
+#             {
+#                 for(int i = _int_0; i < _int_1; i++)
+#                 {
+#                     for(int i = _int_0; i < _int_1; i++)
+#                     {
+#                         for(int i = _int_0; i < _int_1; i++)
+#                         {
+#                             for(int i = _int_0; i < _int_1; i++)
+#                             {
+#                                 for(int i = _int_0; i < _int_1; i++)
+#                                 {
+#                                     for(int i = _int_0; i < _int_1; i++)
+#                                     {
+#                                         for(int i = _int_0; i < _int_1; i++)
+#                                         {
+#                                             for(int i = _int_0; i < _int_1; i++)
+#                                             {
+#                                                 for(int i = _int_0; i < _int_2; i++)
+#                                                 {
+#                                                     for(int i = _int_0; i < _int_1; i++)
+#                                                     {
+#                                                         for(int i = _int_0; i < _int_1; i++)
+#                                                         {
+#                                                             for(int i = _int_0; i < _int_2; i++)
+#                                                             {
+#                                                                 // This inner loop is run four times. The _GLF_global_loop_count
+#                                                                 // changes accordingly: 90, 91, 92, 93.
+#                                                                 do
+#                                                                 {
+#                                                                     _GLF_global_loop_count++;
+#                                                                 }
```

Comment:
It would be clearer to say that on the first iteration, _GLF_global_loop_count is incremented 90 times, and on each of the subsequent 3 iterations it is incremented once, so its value on leaving the loop is 93.

And also to say that the value of f on leaving the loop is 4.0.